### DISCUSSION TITLE
Radius Server unter freetz

### DISCUSSION THREAD

**Discussion by jacoblange78** [Freetz-NG/freetz-ng#568] [Unerfüllte Wünsche]
Hallo,
ich habe heute erfolgreich mein FritzBox 7390 mit dem Freetz OS bestückt, da ich in mehreren Foren gelesen hatte, das es unter Freetz kein Problem ist einen Radius Server für die Authentifizierung der W-Lan Nutzer an zu geben. 

Aber nach Stunden der Internetrecherche und dem durchgucken aller Pakete für Freetz bin ich leider nicht fündig geworden.

Mein Ziel ist es, das alle Wlan Nutzer sich zuerst authentifizieren müssen und da dachte ich, bietet sich doch ein Radius Server an.

Ich würde mich freuen, wenn jemand eine Idee hat, wie ich das Problem mit meiner Box und Freetz lösen könnte.

**Comment by fda77** [comment]
Vieles von den WLAN-Schachen sind bei AVM closed-source. Ob man da einen Radius-Server dazu packen kann weiss ich nicht. Das würde am besten einfach mal jemand ausprobieren

**Comment by cachius** [comment]
Auf den FritzBoxen gibt es standardmäßig eine [angepasste Version von hostapd](https://boxmatrix.info/wiki/Property:hostapd_(lincmd)). Die Symbole sehen vielversprechend aus, es gibt etliche mit `radius_, eap_, ieee802_1x`-Präfix. Allerdings ist [fragwürdig](https://www.ip-phone-forum.de/threads/802-1x-protokoll-auf-fritzbox-7170.300448/#post-2289220 "802.1x Protokoll auf Fritzbox 7170"), ob es mit den AVM-angepassten Versionen funktioniert und wenn ja, wie man die geänderte Konfiguration persistiert:

> es gibt praktisch keinerlei Möglichkeiten für dauerhafte Eingriffe mehr

Freetz  wird nur für den SSH-Zugang zur Fritzbox gebraucht, um Konfigurationen zu testen, nicht für ein extra Paket.
